lib/routes/app_navigator.dart

```diff
@@ -1,12 +1,11 @@
 import 'package:get/get.dart';
 
-class CustomNavigator {
+class AppNavigator {
   void toNamed(String route) {
     Get.toNamed<dynamic>(route);
   }
```

Comment:
Again, what is the point of this class. I can't quite understand :( Do you think we can remove it?

Reply:
I think this class could be used as a wrapper. Like what you said in the other comment, we could pass an enum instead of string to the function parameter since string is prone to errors

Reply:
Yeah but for now, it seems that even as a wrapper, it is really useless :/

Reply:
Yeah hahaha, I think can just keep it for future iteration -- which perhaps will use enum instead of string to make the wrapper useful